types/d3-scale-chromatic/index.d.ts

```diff
@@ -45,173 +48,269 @@ export const schemeSet3: string[];
 // Diverging
 // -----------------------------------------------------------------------
 /**
+ * Diverging color schemes are available as continuous interpolators (often used with d3.scaleSequential) and as discrete schemes
+ *  (often used with d3.scaleOrdinal). Each discrete scheme, such as d3.schemeBrBG, is represented as an array of arrays of hexadecimal
+ *  color strings. The kth element of this array contains the color scheme of size k; for example, d3.schemeBrBG[9] contains an array
+ *  of nine strings representing the nine colors of the brown-blue-green diverging color scheme. Diverging color schemes support
+ *  a size k ranging from 3 to 11.
```

Comment:
In editors that support autocompletion, this information is attached only to the `interpolateBrBG` function. I suggest removing it, save for the information about the _k_ variable, which should get repeated for all of the `const scheme*` variables.